aws/modules/register_group/dns.tf

```diff
@@ -1,11 +0,0 @@
-resource "aws_route53_record" "multi" {
-  count = "${length(var.registers)}"
-  zone_id = "${var.dns_zone_id}"
-  name = "${element(var.registers, count.index)}.${var.vpc_name}.${var.dns_domain}"
-  type = "A"
-  alias {
-    name = "${aws_elb.load_balancer.dns_name}"
-    zone_id = "${aws_elb.load_balancer.zone_id}"
-    evaluate_target_health = false // we don't have anything to fail over to at the moment
-  }
-}
```

Comment:
can we remove the `enable_availability_checks` and `registers` variables from variables.tf here, too?